src/datatypes/clinical.coffee

```diff
@@ -28,18 +32,22 @@ toCodeList = (c) ->
     list
   else if typeIsArray c.codes
     c.codes
-  else if typeof c is 'string'
-    [new Code(c)]
   else
     [c]
 
 
 codesInList = (cl1, cl2) ->
-  cl1.some (c1) -> (cl2.some (c2) -> codesMatch(c1, c2))
+  cl1.some (c1) -> (cl2.some (c2) ->
+    if typeof c1 is 'string'
+      # for "string in codesystem" or "string in valueset" this should compare the string to
+      # the code's "code" field according to the specification.
+      c1 == c2.code
```

Comment:
Is it possible for `cl2` to ever contain a `string`?  If so, we need to account for that, right?  And we probably also need:
```
else if typeof c2 is 'string'
  c2 == (if typeof c1 is string then c1 else c1.code)
```
(Beware: it's been a while since I wrote coffee -- syntax may be wrong).

Reply:
I just checked.  As the code is currently written it is _not_ possible for `cl2` to contain a string.  Could you add a comment (maybe above the function) indicating that `cl1` may contain`Code`s or `string`s, but that `cl2` can only include `code`s?  Either that or update the logic to allow strings on either side...